packages/contracts/src.ts/index.ts

```diff
@@ -913,12 +897,12 @@ export class BaseContract {
 
             // Listen for any event
             if (eventName === "*") {
-                return this._normalizeRunningEvent(new WildcardRunningEvent(this.address, this.interface));
+                return this._normalizeRunningEvent(new WildcardRunningEvent(this._address.toString(), this.interface));
```

Comment:
Shouldn't we cast the `AccountLike` to `solidity address`? What if `address` is `0.0.1`?